src/domains/ecommerce-common/00_alerts.tf

```diff
@@ -254,4 +254,49 @@ AzureDiagnostics
     operator  = "GreaterThanOrEqual"
     threshold = 2
   }
+}
+
+# eCommerce user stats service availability
+resource "azurerm_monitor_scheduled_query_rules_alert" "ecommerce_user_stats_service_availability_alert" {
+  count = var.env_short == "p" ? 1 : 0
+
+  name                = "ecommerce-user-stats-service-availability-alert"
+  resource_group_name = azurerm_resource_group.rg_ecommerce_alerts[0].name
+  location            = var.location
+
+  action {
+    action_group           = [data.azurerm_monitor_action_group.email.id, data.azurerm_monitor_action_group.slack.id]
```

Comment:
Include more detailed information in the alert payload to help the team quickly understand the issue.

```
action {
  action_group           = [data.azurerm_monitor_action_group.email.id, data.azurerm_monitor_action_group.slack.id]
  email_subject          = "[eCommerce] User stats service availability less than 99%"
  custom_webhook_payload = jsonencode({
    "alertName" : "ecommerce-user-stats-service-availability-alert",
    "severity" : "1",
    "description" : "eCommerce User stats service availability less than 98% in the last 30 minutes detected",
    "affectedService" : "User Stats Service",
    "environment" : var.env_short
  })
}

`

Reply:
In all other actions the custom_webhook_payload property is always populated with _“{}”_, so it is most likely not needed.

Reply:
I will analyze the custom_webhook_payload  property and we could make a test with this value populated

Reply:
added here 5544ec956736bec97bd63d4112ae48e0eed9c123